src/test/java/com/mercadopago/resources/mocks/request/payment/payment_base.json

```
@@ -63,5 +63,25 @@
   "payment_method_id": "master",
   "token": "bf9edf6ffae3ab5742033f33c557d54e",
   "transaction_amount": 100,
-  "statement_descriptor": "statementDescriptor"
-}
\ No newline at end of file
+  "statement_descriptor": "statementDescriptor",
+  "payment_method": {
+    "data": {
+      "rules": {
+        "discounts":[{
+          "type":"fixed",
+          "value":5,
+          "limit_date":"2022-10-25"
+        }],
+        "fine":{
+          "type":"fixed",
```

`fixed` não é suportado para fine
```suggestion
          "type":"percentage",
```